monorepo/website/src/pages/docs/how-to/approve-submissions.mdx

```diff
@@ -46,4 +46,57 @@ You can also always view your released sequences by going to 'Browse' in the top
 
 # API
 
-_Instructions coming soon. See the [Swagger API documentation](/api-documentation)_
+The following API requests all require an authentication token. Please read [Authenticating via API guide](/docs/how-to/authentication-api) for the instructions to obtain the token an include the token in the HTTP header `Authorization: Bearer <authentication token>`.
+
+You can retrieve a list of uploaded but not released sequences by sending a GET request to the endpoint `https://backend.pathoplexus.org/<organism>/get-sequences?groupIdsFilter=<group id>&statusesFilter=RECEIVED&statusesFilter=IN_PROCESSING&statusesFilter=HAS_ERRORS&statusesFilter=AWAITING_APPROVAL&warningsFilter=INCLUDE_WARNINGS` where the available values for the organism are: `cchf`, `ebola-sudan`, `ebola-zaire` and `west-nile`. The `sequenceEntries` field of the returned object contains a list of sequences with their corresponding `status`:

```

Comment:
This line is so so long, it should probably be in a little code box on its own. Otherwise very hard to tell apart from the text-text.

Reply:
Thanks, fixed!